### ISSUE TITLE
Unresolved external symbols "uv_close" and "uv_is_closing" on Windows using Vcpkg

### ISSUE THREAD

**Issue by WesThorburn** [uWebSockets/uWebSockets#518]
I've used Vcpkg to install uWebSockets and all its dependencies. Upon building the solution, I'm getting the following three Link errors.
```
1>ConsoleApplication2.obj : error LNK2019: unresolved external symbol "void __cdecl uv_close(struct uv_poll_s *,void (__cdecl*)(struct uv_handle_s *))" (?uv_close@@YAXPEAUuv_poll_s@@P6AXPEAUuv_handle_s@@@Z@Z) referenced in function "public: void __cdecl uS::Socket::close(void)" (?close@Socket@uS@@QEAAXXZ)
1>ConsoleApplication2.obj : error LNK2019: unresolved external symbol "void __cdecl uv_close(struct uv_timer_s *,void (__cdecl*)(struct uv_handle_s *))" (?uv_close@@YAXPEAUuv_timer_s@@P6AXPEAUuv_handle_s@@@Z@Z) referenced in function "public: void __cdecl uS::Socket::cancelTimeout(void)" (?cancelTimeout@Socket@uS@@QEAAXXZ)
1>ConsoleApplication2.obj : error LNK2019: unresolved external symbol "bool __cdecl uv_is_closing(struct uv_poll_s *)" (?uv_is_closing@@YA_NPEAUuv_poll_s@@@Z) referenced in function "public: bool __cdecl uS::Socket::isClosed(void)" (?isClosed@Socket@uS@@QEAA_NXZ)
```
My machine is running Windows 7 64bit, Vcpkg installs libuv v1.10.1 and uWebSockets v0.13.0.
I thought the error may have been with libuv, but after opening [an issue](https://github.com/libuv/libuv/issues/1241),  I was advised the error is more likely with uWebSockets.

What am I missing here? Are there any additional dependency links that I need to configure?

**Comment by WesThorburn** [comment]
I thought that maybe the problem was occurring because Vcpkg was using uWebSockets v0.12.0 at first, and Vcpkg was very recently updated to support uWebSockets v0.13.0. Removing the old uWebSockets version from Vcpkg manually wasn't working, so I removed Vcpkg entirely and reinstalled a fresh copy of it. Using Vcpkg, I installed fresh copies of each of uWebSockets dependencies and fortunately, I'm getting a different error message. Now I'm getting:
```
Cannot open include file: 'uUV.h': No such file or directory
```
The line of code in question is at c:\src\vcpkg\installed\x64-windows\include\uws\networking.h Line 55.

**Comment by WesThorburn** [comment]
The exact same errors are occurring when I clone the uWebSockets repo and build the project as per the installation instructions in the readme, so I don't think it's an issue with vcpkg.

**Comment by alexhultman** [comment]
You are not linking to libuv

**Comment by WesThorburn** [comment]
I've got libuv installed and I've also added 'libuv.lib' to the Additional Dependencies in the VS2015 linker. I've tried placing `#include "uv.h"` before the uWS\uWS.h include but I'm still getting the same errors. Is there some Windows-specific link to libuv that i'm missing?

I've heard such good things about uWebSockets, and I can't wait to get it working.

**Comment by WesThorburn** [comment]
I spent a couple of hours installing and setting up Ubuntu on my dev machine. I installed the uWebSockets dependencies with Linuxbrew, and followed the instructions in the readme to install uWebSockets. I was able to compile the testServer.cpp file using `g++ -std=c++11 testServer.cpp -luWS -lz -lssl` and now I finally have a basic uWS server up and running!

I'm just going to continue with the uWebSockets project(s) using Ubuntu, I've given up on Windows.
